### PR TITLE
feat(1818): sales table for better engagement

### LINKED ISSUE #1818: Sales feed for better engagement
We would need to think about feed where people could learn on most recent activity of stuff to place their buys

### Weights we can choose to the feed
- emotes per nft
- buys per collection
- owners (raising?)
- assets (higher better)

### Table
What should look like and include table

| Name (link to gallery item detail) | Collection (link to collection) | Buyer (link to buyer) |  Relative Date (1 day ago)  | Price of sale (link to TX extrinsic) | 
| --- | --- | --- | --- | --- |

 
Date - on hover it should have shown exact time like **15/01/2022, 06:22**
Toggle to see pictures of NFT item

### What's need TBD
- write resolver to https://github.com/kodadot/rubick/

### Maybe related 
- #744 
- https://github.com/kodadot/nft-gallery/issues/1496
- https://moonbeans.io/#/trades
- https://kanaria.rmrk.app/sales
- https://www.mouse-dao.com/

### PR DESCRIPTION
**Pull request by KngZhi**
- feat: page for sale engagement
- feat: saels feed query
- feat: init page for sales
- refactor: update time format
- refactor: using basic image
- feat: popup for larger nft image

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1818
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

https://user-images.githubusercontent.com/11361147/168977069-7b0001b9-cfca-4a2f-97fc-84d920dbfd29.mov

### CODE REVIEW

File: pages/sales.vue
Comment (kkukelka) [inline pages/sales.vue:25]:
```suggestion
```

File: pages/sales.vue
Comment (kkukelka) [inline pages/sales.vue:28]:
```suggestion
    const title = 'Latest KUSAMA Sales'
```

File: pages/sales.vue
Comment (kkukelka) [inline pages/sales.vue:32]:
```suggestion
      description: 'Featuring the most recently sold NFTs on RMRK.',
```

File: langDir/en.json
Comment (kkukelka) [inline langDir/en.json:529]:
```suggestion
    "subtitle": "Featuring the most recently sold NFTs on RMRK."
```

File: components/sales/SalesTable.vue
Comment (kkukelka) [inline components/sales/SalesTable.vue:124]:
```suggestion
  protected parseDate(ts: number): string {
```

File: components/sales/SalesTable.vue
Comment (kkukelka) [inline components/sales/SalesTable.vue:118]: could you refactor this to RowSales ?

File: components/sales/SalesTable.vue
Comment (kkukelka) [inline components/sales/SalesTable.vue:189]: I know that its probably done the same in SeriesTable but we should really worry about polluting global namespace with all this stuff.

Comment (kkukelka) [reply components/sales/SalesTable.vue:189]: maybe put all the global stuff from SeriesTabel, Salestable, etc into own scss file. but can be done in followup task

File: components/shared/view/BasicPopup.vue
Comment (kkukelka) [inline components/shared/view/BasicPopup.vue:23]:
```suggestion
  @Prop({ type: String, default: 'bottom' }) placement?: string
```

**Review by kkukelka — CHANGES_REQUESTED** [review]
very cool addition. Just a few comments & requests

File: components/sales/SalesTable.vue
Comment (KngZhi) [reply components/sales/SalesTable.vue]: hm, maybe scope first, and then I would extract that SCSS into one file for those stats pages.

I would prefer no global stuff since it would unconsciously break something.

Comment (KngZhi) [reply components/sales/SalesTable.vue]: FYI, For now, I just converted it as scoped. After some PR of stats merged, I would start refactoring.

**Review by kkukelka — APPROVED** [review]

**Comment by kkukelka** [comment]
lgtm, cant merge yet though because of merge conflict ☹️

**Comment by kkukelka** [comment]
pay 500

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $500 @ 74.68 USD/KSM ~ 6.695 $KSM
🧗 [DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc](https://kusama.subscan.io/account/DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)
🔗 [0x4c7928d8c9f2701a557e59701ab05c76aef7323eb4f011709892d55dfa10b158](https://kusama.subscan.io/extrinsic/0x4c7928d8c9f2701a557e59701ab05c76aef7323eb4f011709892d55dfa10b158)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**
